conda-recipe/meta.yaml

```diff
@@ -0,0 +1,34 @@
+{% set data = load_setup_py_data() %}
+
+package:
+  name: covid-if-annotations
+  version: 0.0.3dev1
+
+source:
+  path: ..
+
+build:
+  number: 0
+  skip: True  # [py<36]
+  entry_points:
+    {% for ep in data['entry_points']['console_scripts'] %}
+    - {{ ep }}
+    {% endfor %}
+requirements:
+  build:
+    - python >=3.6
+    - pip
+    # can be removed if bumpversion PR is merged
+    - setuptools_scm
```

Comment:
I still see this in the diff. Did something go wrong in the rebase or am I missing something?

Reply:
you might have to click on "refresh" on the top of the "files changed" page

Reply:
No, I think it's still there:
https://github.com/k-dominik/covid-if-annotations/blob/add-conda-recipe/conda-recipe/meta.yaml#L21-L22

Reply:
sorry - that was my mistake. Doing to many things in parallel I have only checked setup.py for the setuptools_scm remainders. Done!